infinispan-remote/cross-site-replication/docker-compose/docker-compose.yaml

```diff
@@ -9,38 +9,42 @@ services:
     ports:
       - "11222:11222"
     container_name: ispn-lon-1
+    environment:
+      USER: admin
+      PASS: password
     networks:
       - mynetwork
-    volumes:
-      - ${PWD}:/user-config
-    command: -b SITE_LOCAL -c /user-config/infinispan-xsite.xml -Djgroups.cluster.mcast_addr=228.6.7.10 -Drelay.site=LON
+    command: -c infinispan-xsite.xml -Dinfinispan.site.name=LON -Djgroups.cluster.mcast_addr=228.6.7.10 -Djgroups.mcast_port=46656
```

Comment:
the property `jgroups.cluster.mcast_addr` can be removed since it is not defined/read anywhere

Reply:
works well